extensions/charts/cisbenchmark.go

```diff
@@ -0,0 +1,298 @@
+package charts
+
+import (
+	"context"
+	"fmt"
+
+	catalogv1 "github.com/rancher/rancher/pkg/apis/catalog.cattle.io/v1"
+	"github.com/rancher/shepherd/clients/rancher"
+	"github.com/rancher/shepherd/clients/rancher/catalog"
+	"github.com/rancher/shepherd/extensions/defaults"
+	kubenamespaces "github.com/rancher/shepherd/extensions/kubeapi/namespaces"
+	"github.com/rancher/shepherd/extensions/namespaces"
+	"github.com/rancher/shepherd/pkg/api/steve/catalog/types"
+	"github.com/rancher/shepherd/pkg/wait"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+const (
+	CISBenchmarkNamespace = "cis-operator-system"
+	CISBenchmarkName      = "rancher-cis-benchmark"
+	CISBenchmarkCRDName   = "rancher-cis-benchmark-crd"
```

Comment:
Nit; these need comments, examples should be in other charts files. These are going to be exposed with Godoc.

Reply:
Sure, I'll add them.

Reply:
Resolved